.gitignore

```diff
@@ -8,4 +8,5 @@ build
 dist
 server-config.js
 .idea
-.idea/*
\ No newline at end of file
+.idea/*
+/.idea/mosaico.iml
```

Comment:
To ignore the .idea folder that PHPStorm/IntelliJ IDEA creates we just need a single line:
.idea/

This will ignore the folder and its contents.